docs/source/api/plugin/landing-pages.mdx

```diff
@@ -461,94 +461,7 @@ The default value is `dark`.
-
-The `ApolloServerPluginLandingPageGraphQLPlayground` plugin serves the [GraphQL Playground IDE](https://github.com/graphql/graphql-playground) as a landing page.
-
-The GraphQL Playground plugin lives in its own `@apollo/server-plugin-landing-page-graphql-playground` package. To use it, install this package and provide the plugin to the `ApolloServer` constructor:
-
-<MultiCodeBlock>
-
-```ts
-import { ApolloServer } from '@apollo/server';
-import {
-  ApolloServerPluginLandingPageGraphQLPlayground
-} from "@apollo/server-plugin-landing-page-graphql-playground";
-
-const server = new ApolloServer({
-  typeDefs,
-  resolvers,
-  plugins: [
-    ApolloServerPluginLandingPageGraphQLPlayground(),
-  ],
-});
-```
-
-</MultiCodeBlock>
-
-This plugin takes the same arguments as the [`renderPlaygroundPage` function](https://github.com/apollographql/graphql-playground/blob/apollo/packages/graphql-playground-html/src/render-playground-page.ts#L63) from the `graphql-playground-html` package (or specifically, the `@apollographql/graphql-playground-html` fork).
-
-[The table below](#options-2) mentions a few of the more common options and is not exhaustive.
-
-### Options
-
-<table class="field-table">
-  <thead>
-    <tr>
-      <th>Name /<br/>Type</th>
-      <th>Description</th>
-    </tr>
-  </thead>
-
-<tbody>
-
-<tr>
-<td>
-
-###### `version`
-
-`string`
-</td>
-<td>
-
-By default, this plugin loads a specific npm version of `@apollographql/graphql-playground-react` from a CDN. The version is hard-coded in `apollo-server-core` and is typically incremented when new versions of the Playground fork are released (this is rare because the project is retired).
-
-You can specify a different version here.
-
-</td>
-</tr>
-
-<tr>
-<td>
-
-###### `endpoint`
-
-`string`
-</td>
-<td>
-
-By default, GraphQL Playground connects to a GraphQL server hosted at the same URL as Playground itself. To specify a different GraphQL endpoint, use this option.
-
-</td>
-</tr>
-
-<tr>
-<td>
-
-###### `settings`
-
-`Object`
-</td>
-<td>
-
-If specified, allows you to override the default values of GraphQL Playground's settings. Documentation of supported settings is available in [GraphQL Playground's README](https://github.com/graphql/graphql-playground#settings).
-
-</td>
-</tr>
-
-
-</tbody>
-</table>
+Apollo Server 3 provided a GraphQL Playground landing page by default. For migration purposes, we have published the [`@apollo/plugin-landing-page-graphql-playground`](https://www.npmjs.com/package/@apollo/plugin-landing-page-graphql-playground) package, which is a GraphQL Playground plugin that is compatible with Apollo Server 4. However, we do not intend to support this plugin via documentation, security updates, or otherwise. Instead, we recommend migrating to the actively maintained Apollo Sandbox (the default landing page in Apollo Server 4) at your earliest convenience.
```

Comment:
I thought AS2 used the GraphQL Playground landing page by default? Did AS3 as well? A few tweaks:
 
```suggestion
By default, Apollo Server 3 provided a GraphQL Playground landing page. For migration purposes, we've published the [`@apollo/plugin-landing-page-graphql-playground`](https://www.npmjs.com/package/@apollo/plugin-landing-page-graphql-playground) package, a GraphQL Playground plugin compatible with Apollo Server 4. However, we aren't supporting this plugin with documentation or security updates. Instead, we recommend migrating to the actively maintained [Apollo Sandbox](/studio/explorer/sandbox) (the default landing page in Apollo Server 4) at your earliest convenience.
```